cmd/http-tracer.go

```diff
@@ -80,6 +81,17 @@ func (r *recordRequest) Data() []byte {
 	return logger.BodyPlaceHolder
 }
 
+var ldapPwdRegex = regexp.MustCompile("(^.*?)LDAPPassword=([^&]*?)(&(.*?))?$")
+
+// redact LDAP password if part of string
+func redactLDAPPwd(s string) string {
+	if ldapPwdRegex.MatchString(s) {
```

Comment:
This is not needed, you can check for len(parts) to check if regex found a result or not.